docs/tutorials/qsimcirq.ipynb

```diff
@@ -1,87 +1,139 @@
       "metadata": {
-        "id": "vAx5g2rPnlzt",
-        "colab_type": "text"
+        "colab_type": "text",
+        "id": "8M2lhf2RBGZj"
       },
       "source": [
-        "# Getting started with qsimcirq\n",
-        "\n",
-        "The qsim library provides a Python interface to Cirq in the **qsimcirq** PyPI package."
+        "##### Copyright 2020 Google"
       ]
     },
     {
       "cell_type": "code",
+      "execution_count": null,
       "metadata": {
-        "id": "XcAhsIQdivnp",
+        "cellView": "form",
+        "colab": {},
         "colab_type": "code",
-        "colab": {
-          "base_uri": "https://localhost:8080/",
-          "height": 17
-        },
-        "outputId": "2a1f75c0-182c-47a6-83fe-83b0f7aaeb83"
+        "id": "FMqHt2UBBJzR"
+      },
+      "outputs": [],
+      "source": [
+        "#@title Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+        "# you may not use this file except in compliance with the License.\n",
+        "# You may obtain a copy of the License at\n",
+        "#\n",
+        "# https://www.apache.org/licenses/LICENSE-2.0\n",
+        "#\n",
+        "# Unless required by applicable law or agreed to in writing, software\n",
+        "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+        "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+        "# See the License for the specific language governing permissions and\n",
+        "# limitations under the License."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "vAx5g2rPnlzt"
+      },
+      "source": [
+        "# Get started with qsimcirq"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "5jqw8k0zBsb-"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.example.org/qsim/tutorials/qsimcirq\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on QuantumLib</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/quantumlib/qsim/blob/master/docs/tutorials/qsimcirq.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/quantumlib/qsim/blob/master/docs/tutorials/qsimcirq.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/qsim/docs/tutorials/qsimcirq.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
```

Comment:
These links don't currently work, but I guess they will once this PR is submitted?

Reply:
Correct. Once this is merged the Colab and GitHub links will work.
The Download button (`storage.googleapis.com`) will work after the first site import (non-preview).